creator-node/src/config.js

```diff
@@ -254,6 +254,30 @@ const config = convict({
     env: 'debounceTime',
     default: 30000 // 30000ms = 30s
   },
+  dataRegistryAddress: {
+    doc: 'data contracts registry address',
+    format: String,
+    env: 'dataRegistryAddress',
+    default: null
+  },
+  dataProviderUrl: {
+    doc: 'data contracts web3 provider url',
+    format: String,
+    env: 'dataProviderUrl',
+    default: null
+  },
+  dataNetworkId: {
+    doc: 'data contracts network id',
+    format: String,
+    env: 'dataNetworkId',
+    default: null
+  },
+  creatorNodeEndpoint: {
```

Comment:
@vicky-g i don't think we need this right now for either:
- libs init with data contracts
- enforcement of owner wallet + private key

can you confirm @SidSethi?

Reply:
yep we def don't need it for rest of this code change, you think better to remove for this PR?

Reply:
aren't these configs used to setup web3Config? 
https://github.com/AudiusProject/audius-protocol/pull/535/files#diff-82266090da321e8d0273a1976d919014R40-R43

Reply:
@vicky-g i think we didn't specifically need `creatorNodeEndpoint`. everything else, we do

Reply:
ah, got it! will remove